internal/service/catchall/ephemeral.go

```diff
@@ -76,16 +76,26 @@ func (m *MBOPServer) findUsersBy(accountNo string, orgID string, adminOnly strin
 
 	out := []models.User{}
 	for _, user := range usersList {
-		// When adminOnly is true, parameter “status” is ignored
-		if adminOnly == "true" && !user.IsOrgAdmin {
-			continue
+
+		if adminOnly != "" {
+			onlyAdmin, err := strconv.ParseBool(adminOnly)
```

Comment:
nice switch to convert to a bool, makes the following checks using the `onlyAdmin` variable cleaner to read imo :cowboy_hat_face: 

one little nit pick I would say is maybe name it `isAdminOnly` to signify it is now a bool, instead of `onlyAdmin`